### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by developerdenesh**
Adding Globotix to the list of companies

### CODE REVIEW

**Comment by vmayoral** [comment]
This is quite unique, thanks @developerdenesh. Could you share how are you related to the company? Also, any chance to add a few bits of ROS usage? (version, packages used, etc.)

**Comment by developerdenesh** [comment]
Thanks! Sure - currently a software engineer here. Our stack is largely built on top of ROS 1, and making use of many of the "core" packages such as move_base

File: README.md [hunk 1]
```diff
@@ -125,6 +125,7 @@ A public list of companies that are known to use the Robot Operating System (ROS
...
 | [Glidewell Laboratories](https://glidewelldental.com/) |  Manufacture dental implants and tooth replacement systems. The company is [currently member](https://rosindustrial.org/ric/current-members/#glidewell-laboratories) of ROS-Industrial, is an open-source project that extends the advanced capabilities of ROS to manufacturing automation and robotics (see [`ros-industrial`](https://github.com/ros-industrial))[^138]. | 1970 | 
+| [Globotix](https://globotix.sg/) | Facilities Management robotics solution focused on delivering customised solutions tailored to the needs of clients. Its unique modular system allows for multiple functionality to be integrated into the same base system whilst providing flexibility for the client to choose the best suite of tools suited to their requirements. | 2016 |
```
Comment (vmayoral) [inline README.md:128]:
```suggestion
| [Globotix](https://globotix.sg/) | Facilities Management robotics solution focused on delivering customised solutions tailored to the needs of clients. Its unique modular system allows for multiple functionality to be integrated into the same base system whilst providing flexibility for the client to choose the best suite of tools suited to their requirements. Stack is largely built on top of ROS 1, and making use of many of the *core* packages such as `move_base`  | 2016 |
```

**Comment by vmayoral** [comment]
Thanks @developerdenesh, added those bits to your commit. Good to go!

**Pull request merged**